### PR TITLE
Branding abstraction

### PR DESCRIPTION
**Pull request by ezwang**
Abstract all references to the Penn Clubs logo, "Penn Clubs", "clubs", "student organizations", and "University of Pennsylvania" into a `branding.tsx` file that we can use to easily change branding globally throughout the site.

Will be used with our First Year Hub initiatives and potentially allow the Penn Clubs codebase to be used for other schools and purposes.

- There are singular and plural versions for nouns describing an organization on the site.
- Don't rebrand the FAQ page, this will be replaced entirely.
- Could be worth to try to make some very simple tests (e.g. grep Penn Clubs in various folders) to make sure we don't introduce naming issues in a later commit.

### CODE REVIEW

**Comment by cphalen** [comment]
lol we abstracted Penn Clubs out of Penn Clubs. The changes look good 👍

File: frontend/components/common/AuthPrompt.tsx [hunk 1]
```diff
@@ -46,13 +47,13 @@ const AuthPrompt = ({
...
         <TitleHeader>
-          <Image src="/static/img/peoplelogo.png" />
+          <Image src={SITE_LOGO} />
           <Title>{title}</Title>
         </TitleHeader>
         <Margin>
           <Text>
             {children ||
-              'To make the most of Penn Clubs features, like bookmarking and subscribing to clubs, please login using your PennKey.'}
+              `To make the most of ${SITE_NAME} features, like bookmarking and subscribing to clubs, please login using your PennKey.`}
```
Comment (davish) [inline frontend/components/common/AuthPrompt.tsx:56]:
```suggestion
              `To make the most of ${SITE_NAME} features, like bookmarking and subscribing to {OBJECT_NAME_PLURAL}, please login using your PennKey.`}
```

**Review by davish — COMMENTED** [review]
Found one miss that I wanted to make sure you're aware of.

Also, am wondering about color scheme. I'm guessing FYH will have a different one than clubs (red+blue, etc), and wondering how that would be handled?

**Comment by ezwang** [comment]
> Found one miss that I wanted to make sure you're aware of.

@davish nice catch!

> Also, am wondering about color scheme. I'm guessing FYH will have a different one than clubs (red+blue, etc), and wondering how that would be handled?

@snie11 is going to create a new color scheme and we'll override the color variables in the constants.

**Pull request merged**